external-crates/move/crates/move-compiler/src/to_bytecode/translate.rs

```diff
@@ -1077,3 +1082,31 @@ fn binary_op(code: &mut IR::BytecodeBlock, sp!(loc, op_): BinOp) {
         },
     ));
 }
+
+fn abort_code(context: &mut Context, code: &mut IR::BytecodeBlock, ecode: H::Exp) {
+    use IR::Bytecode_ as B;
+    match ecode.exp {
+        sp!(cloc, H::UnannotatedExp_::Constant(nm))
+            if context.current_module().is_some_and(|m| {
+                context.constant_attributes(m, nm).is_some_and(|attrs| {
+                    attrs.contains_key_(&known_attributes::ErrorAttribute.into())
+                })
+            }) =>
+        {
+            let line_no = context.env.file_mapping().location(cloc).start.line;
+
+            // Clamp line number to u16::MAX -- so if the line number exceeds u16::MAX, we don't
+            // record the line number essentially.
+            let line_number = std::cmp::min(line_no, u16::MAX as usize) as u16;
```

Comment:
I honestly think it's fine to do this since I think it's a bit odd to have a "hard" restriction on file size imposed by this, and I've actually come around to clamping it and at most maybe issuing a warning here?